Jenkinsfile-subsquid

```diff
@@ -68,16 +64,19 @@ pipeline {
                             apt-get install make git curl -y
```

Comment:
i think that creating new container for every stage is redundant, it takes time and resources. Also no need to install software on every step, move it to separate step

Reply:
This is a separate stage, depending what developers need

Reply:
you can run the whole pipeline in this container and install deps in the first separate stage